apps/meteor/client/views/omnichannel/directory/calls/CallTable.tsx

```diff
@@ -54,6 +55,17 @@ const CallTable: FC = () => {
 	const query = useQuery(debouncedParams, debouncedSort, userIdLoggedIn);
 	const directoryRoute = useRoute('omnichannel-directory');
 
+	const resolveDirectionLabel = useCallback(
+		(direction: IVoipRoom['direction']) => {
+			const labels: { [key: string]: 'Incoming' | 'Outgoing' } = {
```

Comment:
Actually, here you could use `as const`. Ex: 
```
const label = {
 ...
} as const
```

Reply:
That's a really good suggestion. Thanks!!